app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -63,15 +63,35 @@ class RefereeProfile extends Component {
     } = this.state
 
     return {
-      firstName: changedFirstName || referee.firstName,
-      lastName: changedLastName || referee.lastName,
+      firstName: changedFirstName !== null ? changedFirstName : referee.firstName,
+      lastName: changedLastName !== null ? changedLastName : referee.lastName,
       nationalGoverningBodies: changedNGBs || referee.nationalGoverningBodies,
-      bio: changedBio || referee.bio,
+      bio: changedBio !== null ? changedBio : referee.bio,
       showPronouns: changedShowPronouns || referee.showPronouns,
-      pronouns: changedPronouns || referee.pronouns
+      pronouns: changedPronouns !== null ? changedPronouns : referee.pronouns
     }
   }
 
+  get refereeDisplayName() {
+    const {
+      referee: { firstName, lastName }
+    } = this.state
+
+    if (!firstName && !lastName) {
+      return 'Anonymous Referee'
+    }
+
```

Comment:
Future improvement we could make is turn this into a utility function housed in a utilities folder potentially. Because the referee table could probably use something similar, so that we don't need to filter the refs in that table. That said, not a blocker for this PR.